app/helpers/filters_helper.rb

```diff
@@ -19,4 +19,8 @@ def search_path(type)
   def auto_complete_search_path
     '/auto_complete_search'
```

Comment:
Unrelated, but this is so very very wrong. Likely the cause of https://projects.theforeman.org/issues/26437